use-timescale/continuous-aggregates/real-time-aggregates.md

```diff
@@ -9,27 +9,34 @@ import CaggsRealTimeHistoricalDataRefreshes from 'versionContent/_partials/_cagg
 
 # Real time aggregates
 
-Continuous aggregates do not include the most recent data chunk from the
-underlying hypertable. Real time aggregates use the aggregated data and add the
-most recent raw data to it to provide accurate and up to date results, without
-needing to aggregate data as it is being written. In Timescale&nbsp;1.7 and later,
-real time aggregates are enabled by default. When you create a continuous
-aggregate view, queries to that view include the most recent data, even if
-it has not yet been aggregated.
-
-For more detail on the comparison between continuous and real time aggregates,
-see our [real time aggregate blog post][blog-rtaggs].
+<Highlight type="important">
+In Timescale&nbsp;1.7 and later, real time aggregates are enabled by default.
+You do not need to manually enable real-time aggregation.
+</Highlight>
+
+Real-time aggregates use the aggregated data and add the most recent raw data to
```

Comment:
"Real-time aggregates use the aggregated data and add not-yet materialized sections using a live query of the underlying raw data to ..."